packages/components/src/VirtualizedList/VirtualizedList.scss

```diff
@@ -9,20 +9,28 @@
 	height: calc(100% - 2px);
 	padding: 0 0 0 $padding-large;
 	position: relative;
-	overflow: hidden;
-	text-overflow: ellipsis;
-	white-space: nowrap;
 
 	&:hover {
 		cursor: pointer;
 	}
 	:global(.ReactVirtualized__Table__headerTruncatedText) {
 		align-self: center;
-		overflow: hidden;
 		padding-left: $padding-small;
+	}
+}
+
+:global(.ReactVirtualized__Table__headerColumn) {
```

Comment:
👍🏻 that's nice, a great improvement!

You might use a real CSS module instead of using a global style, to be really delightful.